### ISSUE TITLE
Unhandled rejection TypeError: Cannot read property 'injectedData' of undefined

### ISSUE THREAD

**Issue by hjalmers** [jgranstrom/sass-extract#5]
Maybe I'm missing something but isn't sass-extract suppose to work like drop in replacement for node-sass?

The following works as expected for me:
```js
const sass = require('node-sass');
const sassExtract = require('sass-extract');

function extract() {

    const rendered = sass.render({
        file: 'scss/style.scss'
    });
}
```

But if I change from `sass.render` to `sassExtract.render ` like this:
```js
const sass = require('node-sass');
const sassExtract = require('sass-extract');

function extract() {

    const rendered = sassExtract.render({
        file: 'scss/style.scss'
    });
}
```
I get the following error:

```js
Unhandled rejection TypeError: Cannot read property 'injectedData' of undefined
    at makeExtractionCompileOptions (C:\dev\style\packages\colors\node_modules\sass-extract\lib\extract.js:52:61)
    at C:\dev\style\packages\colors\node_modules\sass-extract\lib\extract.js:101:36
    at tryCatcher (C:\dev\style\packages\colors\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\dev\style\packages\colors\node_modules\bluebird\js\release\promise.js:512:31)
    at Promise._settlePromise (C:\dev\style\packages\colors\node_modules\bluebird\js\release\promise.js:569:18)
    at Promise._settlePromise0 (C:\dev\style\packages\colors\node_modules\bluebird\js\release\promise.js:614:10)
    at Promise._settlePromises (C:\dev\style\packages\colors\node_modules\bluebird\js\release\promise.js:693:18)
    at Promise._fulfill (C:\dev\style\packages\colors\node_modules\bluebird\js\release\promise.js:638:18)
    at Promise._resolveCallback (C:\dev\style\packages\colors\node_modules\bluebird\js\release\promise.js:432:57)
    at Promise._settlePromiseFromHandler (C:\dev\style\packages\colors\node_modules\bluebird\js\release\promise.js:524:17)
    at Promise._settlePromise (C:\dev\style\packages\colors\node_modules\bluebird\js\release\promise.js:569:18)
    at Promise._settlePromise0 (C:\dev\style\packages\colors\node_modules\bluebird\js\release\promise.js:614:10)
    at Promise._settlePromises (C:\dev\style\packages\colors\node_modules\bluebird\js\release\promise.js:693:18)
    at Promise._fulfill (C:\dev\style\packages\colors\node_modules\bluebird\js\release\promise.js:638:18)
    at PromiseArray._resolve (C:\dev\style\packages\colors\node_modules\bluebird\js\release\promise_array.js:126:19)
    at PromiseArray._promiseFulfilled (C:\dev\style\packages\colors\node_modules\bluebird\js\release\promise_array.js:144:14)
```
Am I missing something?

**Comment by jgranstrom** [comment]
You are absolutely right. The problem was compatibility issues with Windows. These are now fixed in version 0.4.0.

Can you please try `npm install sass-extract@0.4.0` and see if you still experience any issues?

**Comment by hjalmers** [comment]
Perfect! Can confirm it works like a charm now:D 

Tack så mycket för bra bibliotek och trevlig helg!
